rocky/katalogus/templates/plugin_deep_link_list.html

```diff
@@ -0,0 +1,79 @@
+{% extends "layouts/base.html" %}
+
+{% load static %}
+{% load i18n %}
+
+{% block content %}
+    {% include "header.html" %}
+
+    <main id="main-content">
+        {% include "partials/katalogus_header.html" %}
+        {% include "partials/plugins_navigation.html" with plugin_type="deep_link" %}
+
+        <section>
+            <div>
+                <div class="fifty-fifty">
+                    <div>
+                        <h2>{% translate "Deep Links" %}</h2>
+                        <p>
+                            {% blocktranslate %}
+                                Deep links are links from an external source giving more information about an object.
+                                It could be a wikipedia page, CVE score page or anything that can give extended information about a particular object.
+                            {% endblocktranslate %}
+                        </p>
+                    </div>
+                    <div class="horizontal-view toolbar">
+                        <a class="button"
+                           href="{% url 'plugin_deep_link_add' organization.code %}">{% translate "Create deep link" %}</a>
+                    </div>
+                </div>
+                {% if deep_links %}
+                    <p>
+                        <strong>{{ deep_links|length }}</strong> {% translate "Deep links available" %}
+                    </p>
+                    <div class="horizontal-scroll">
+                        <table>
+                            <caption>{% translate "Overview of deep links:" %}</caption>
+                            <thead>
+                                <tr>
+                                    <th>{% translate "Name" %}</th>
+                                    <th>{% translate "OOI-Type" %}</th>
+                                    <th>{% translate "Actions" %}</th>
+                                </tr>
+                            </thead>
+                            <tbody>
+                                {% for deep_link in deep_links %}
+                                    <tr>
+                                        <td>{{ deep_link.plugin.name }}</td>
```

Comment:
This can also link to the edit form for this link.

Reply:
Added!
<img width="1169" alt="image" src="https://github.com/minvws/nl-kat-coordination/assets/30832687/76b2c00f-90a2-40fa-ad7b-b8a0d5b0a328">